eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/mapper/connection/ConnectionMapper.java

```diff
@@ -34,15 +35,28 @@
 @Mapper
 public interface ConnectionMapper {
 
-    @Select("SELECT * FROM connection WHERE id = #{id}")
-    ConnectionEntity selectById(ConnectionEntity connectionEntity);
-
     @Select("SELECT * FROM connection")
     List<ConnectionEntity> selectAll();
 
     @Select("SELECT * FROM connection WHERE cluster_id = #{clusterId}")
     List<ConnectionEntity> selectByClusterId(ConnectionEntity connectionEntity);
 
+    @Select("SELECT * FROM connection WHERE cluster_id = #{clusterId} AND source_id = #{sourceId} AND source_type = #{sourceType}")
+    public List<ConnectionEntity> selectByClusterIdSourceTypeAndSourceId(ConnectionEntity connectionEntity);
+
+    @Select("SELECT * FROM connection WHERE cluster_id = #{clusterId} AND sink_id = #{sinkId} AND sink_type = #{sinkType}")
+    public List<ConnectionEntity> selectByClusterIdSinkTypeAndSinkId(ConnectionEntity connectionEntity);
+
+    @Select("SELECT * FROM connection WHERE cluster_id = #{clusterId} AND source_id = #{sourceId} AND source_type = #{sourceType} "
+        + "AND create_time > #{startTime} AND create_time < #{endTime}")
+    public List<ConnectionEntity> selectByClusterIdSourceTypeAndSourceIdAndCreateTimeRange(ConnectionEntity connectionEntity,
```

Comment:
Is the method name too long? Maybe you can define the method from bussiness level.

Reply:
I can't find a suitable name, and this method will be encapsulated in the service code, so I think this is OK.

Reply:
alright